### DISCUSSION TITLE
SSL installation failure on antmedia 2.5.3 enterprise version

### DISCUSSION THREAD

**Discussion by hd51515** [ant-media/Ant-Media-Server#5085] [General]
Installed antmedia 2.5.3 EE and the service is active on my local machine. I also have the port 80 enabled on my local machine.

/usr/local/antmedia$ sudo ufw status
Status: active

To                         Action      From
--                         ------      ----
80                         ALLOW       Anywhere                  
80 (v6)                    ALLOW       Anywhere (v6)             

option-1 fails with below errors. There is no apach2 or ngnix running on this machine
sudo service apache2 stop
Failed to stop apache2.service: Unit apache2.service not loaded.

sudo service ngnix stop
Failed to stop ngnix.service: Unit ngnix.service not loaded.

sudo ./enable_ssl.sh 
14:33:06 > Waiting for DNS validation.
DNS success, installing the SSL certificate.
domain: ams-13330.antmedia.cloud
creating new certificate
/usr/lib/python3/dist-packages/requests/__init__.py:80: RequestsDependencyWarning: urllib3 (1.26.3) or chardet (3.0.4) doesn't match a supported version!
  RequestsDependencyWarning)
Saving debug log to /var/log/letsencrypt/letsencrypt.log
Plugins selected: Authenticator standalone, Installer None
Obtaining a new certificate
Performing the following challenges:
http-01 challenge for ams-13330.antmedia.cloud
Waiting for verification...
Cleaning up challenges
Failed authorization procedure. ams-13330.antmedia.cloud (http-01): urn:ietf:params:acme:error:connection :: The server could not connect to the client to verify the domain :: 71.178.183.52: Fetching http://ams-13330.antmedia.cloud/.well-known/acme-challenge/eBgkzRPZHk-XZEd8w8ch4-qXRpUB2VDaK4-BseykOGg: Timeout during connect (likely firewall problem)

IMPORTANT NOTES:
 - The following errors were reported by the server:

   Domain: ams-13330.antmedia.cloud
   Type:   connection
   Detail: 71.178.183.52: Fetching
   http://ams-13330.antmedia.cloud/.well-known/acme-challenge/eBgkzRPZHk-XZEd8w8ch4-qXRpUB2VDaK4-BseykOGg:
   Timeout during connect (likely firewall problem)

   To fix these errors, please make sure that your domain name was
   entered correctly and the DNS A/AAAA record(s) for that domain
   contain(s) the right IP address. Additionally, please check that
   your computer has a publicly routable IP address and that no
   firewalls are preventing the server from communicating with the
   client. If you're using the webroot plugin, you should also verify
   that you are serving files from the webroot path you provided.
There is a problem in installing SSL to Ant Media Server.
 Please take a look at the logs above and try to fix.
 If you do not have any idea, contact@antmedia.io

**Comment by tremzo** [comment]
To get a free SSL certificate like this you'd need to make sure that your router is forwarding port 80 to the machine that's got AMS installed. Otherwise, you'll see some errors. 

Alternatively, you'd probably want to create a self-signed SSL certificate on your local machine rather than trying to reconfigure your router.  

We have this guide on how to create a self-signed SSL certificate for this very purpose https://antmedia.io/docs/Frequently-Asked-Questions/#how-to-use-self-signed-certificate-on-ant-media-server. This is generally what you'd do if you are running Ant Media Server on your local machine.

**Comment by hd51515** [comment]
Thanks - that worked
